test/integration/text.js

```diff
@@ -420,29 +405,6 @@ describe('BatchText', function() {
     console.warn('Api keys are now required. Skipping some tests.\nhttp://docs.indico.io/v2.0/docs/api-keys')
```

Comment:
Probably a separate PR but we should find/replace these links since they're now wrong.